doc/source/train/user-guides/using-gpus.rst

```diff
@@ -104,6 +104,58 @@ You can get a list of associated devices with :meth:`ray.train.torch.get_devices
     trainer.fit()
 
 
+(PyTorch) Setting the communication backend 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+    This is an advanced setting. In most cases, you don't have to change this setting.
+
+You can set the PyTorch distributed communication backend (e.g. GLOO or NCCL) by passing a
+:class:`~ray.train.torch.TorchConfig` to the :class:`~ray.train.torch.TorchTrainer`.
+
+See the `PyTorch API reference <https://pytorch.org/docs/stable/distributed.html#torch.distributed.init_process_group>`__
+for valid options.
+
+.. testcode::
+    :hide:
+
+    num_training_workers = 1
+
+.. testcode::
+
+    from ray.train.torch import TorchConfig, TorchTrainer
+
+    trainer = TorchTrainer(
+        train_func,
+        scaling_config=ScalingConfig(
+            num_workers=num_training_workers,
+            use_gpu=True,
+        ),
+        torch_config=TorchConfig(backend="gloo"),
```

Comment:
```suggestion
        torch_config=TorchConfig(backend="nccl"),
```

https://alband.github.io/doc_view/distributed.html#backends-that-come-with-pytorch

Seems that the rule of thumb is to use the NCCL backend for distributed GPU training. Let's change it to nccl?

Reply:
Oh this section I just moved. The reason why this one says `gloo` is because `nccl` is already the default.